omnibus-sepolia-carina.toml

```diff
@@ -0,0 +1,436 @@
+func = "setMarketConfiguration"
+
+[[invoke.set_global_market_configuration.args]]
+pythPublishTimeMin = 12
+pythPublishTimeMax = 60
+minOrderAge = 12
+maxOrderAge = 60
+minKeeperFeeUsd = "<%= parseEther('15') %>"
+maxKeeperFeeUsd = "<%= parseEther('500') %>"
+keeperProfitMarginUsd = "<%= parseEther('20') %>"
+keeperProfitMarginPercent = "<%= parseEther('0.2') %>"
+keeperSettlementGasUnits = 1200000
+keeperCancellationGasUnits = 600000
+keeperFlagGasUnits = 1200000
+keeperLiquidateMarginGasUnits = 1200000
+keeperLiquidationGasUnits = 1200000
+keeperLiquidationFeeUsd = "<%= parseEther('5') %>"
+keeperLiquidationEndorsed = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9" # TODO, this is currently owner, change to a keeper
+collateralDiscountScalar = "<%= parseEther('1') %>"
+minCollateralDiscount = "<%= parseEther('0.01') %>"
+maxCollateralDiscount = "<%= parseEther('0.05') %>"
+sellExactInMaxSlippagePercent = "<%= parseEther('0.05') %>"
+utilizationBreakpointPercent = "<%= parseEther('0.85') %>"
+lowUtilizationSlopePercent = "<%= parseEther('0.0003') %>"
+highUtilizationSlopePercent = "<%= parseEther('0.015') %>"
+
+# --- Set eth oracle node id --- #
+#
+#
+#
+[invoke.set_eth_oracle_node_id]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setEthOracleNodeId"
+args = ["<%= extras.eth_oracle_id %>"]
+
+# --- Create Market --- #
+#
+#
+#
+[invoke.create_market]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "createMarket"
+args = [{ name = "<%= formatBytes32String('ETHPERP') %>" }]
+extra.eth_market_id.event = "MarketCreated"
+extra.eth_market_id.arg = 0
+
+# --- Set Specific Market Configuration --- #
+#
+#
+
+[invoke.set_market_configuration]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setMarketConfigurationById"
+
+[[invoke.set_market_configuration.args]]
+marketId = "<%= extras.eth_market_id %>"
+oracleNodeId = "<%= extras.eth_oracle_id %>"
+pythPriceFeedId = "0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace" # https://pyth.network/developers/price-feed-ids#pyth-evm-mainnet"
+makerFee = "<%= parseEther('0.0002') %>"
+takerFee = "<%= parseEther('0.0006') %>"
+maxMarketSize = "<%= parseEther('27000') %>"
+maxFundingVelocity = "<%= parseEther('9') %>"
+skewScale = "<%= parseEther('1000000') %>"
+fundingVelocityClamp = "<%= parseEther('00001') %>"
+minCreditPercent = "<%= parseEther('1') %>"
+minMarginUsd = "<%= parseEther('500') %>"
+minMarginRatio = "<%= parseEther('0.05') %>"
+incrementalMarginScalar = "<%= parseEther('1') %>"
+maintenanceMarginScalar = "<%= parseEther('0.5') %>"
+maxInitialMarginRatio = "<%= parseEther('0.9') %>"
+liquidationRewardPercent = "<%= parseEther('0.00002') %>"
+liquidationLimitScalar = "<%= parseEther('1') %>"
+liquidationWindowDuration = 30
+liquidationMaxPd = "<%= parseEther('0.0016') %>"
+
+# --- Configure collateral for BFP --- #
+#
+#
+# TODO figure out how to get sUSD address
```

Comment:
Something like `system.USDProxy.address`? System belongs in core.toml, unsure of the exact cannon reference syntax/path.

Reply:
Yep figures it out!